can_db/e46.md

```diff
@@ -4,24 +4,30 @@ The CAN bus is not available at the OBD-II port, so it must be tapped into behin
 The easiest places to do this are either behind the intrument cluster or at the steering angle sensor, 
 which is located at the base of the steering column near the firewall.
 
+![Steering Angle Sensor Plug](../images/e46_sensor_plug.jpg)
+
 A simple plug-in harness can be made by cutting both sides of this connector and a few inches of wiring 
-from a donor car at the salvage yard.  Constant and switched power can also be found at this connector.
+from a donor car at the salvage yard.  The CAN-H and CAN-L are the Yellow/Black and Yellow/Brown wires, respectively.
+Constant/switched power and ground can also be found at this connector.  The harness pictured does not include the 
+power/ground.  
+
+![CAN bus harness](../images/e46_harness.jpg)
```

Comment:
It's very hard to see the wire colors on that photos, presumably due to incandescent light source?
Can you try to take a photo outside in bright sunlight, or a similar light source?

Also I'd appreciate it if you can shrink/compress the file to be <300 kB.

Reply:
If you don't have GIMP/Photoshop to shrink/compress the photo please let me know and I can help with that.